Sources/SwiftSyntax/TraversalMode.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alex Hoppen on 27.7.22.
```

Comment:
Sorry, but reviewing on my phone.

But is this file used?

Reply:
Oh, that was committed accidentally. It belongs to a different PR. Thanks for catching it.